pcs/prop.py

```diff
@@ -147,12 +147,15 @@ def list_property(lib, argv, modifiers):
     else:
         properties = {}
 
+
     if not modifiers.get("--defaults"):
         configured_properties = utils.get_set_properties(
             None if print_all else argv[0], properties
         )
         if modifiers.get("--all"):
             properties.update(configured_properties)
+        elif not configured_properties:
+            properties = get_default_property(argv[0])
```

Comment:
What is the point of loading properties again? They are already loaded in the `properties` variable.